docs/pages/access-controls/access-graph/self-hosted-helm.mdx

```diff
@@ -0,0 +1,220 @@
+---
+title: Run Teleport Access Graph on Self-Hosted Clusters with Helm
+---
+
+Using Access Graph with a self-hosted Teleport cluster requires
+setting up the Teleport Access Graph (TAG) service.
+TAG is a dedicated service which uses PostgreSQL as its backing storage
+and communicates with Auth Service and Proxy Service
+to collect information about resources and access.
+
+This guide will help you set up the TAG service
+in a Kubernetes cluster using a Helm Chart,
+and enable the Access Graph feature in your Teleport cluster.
+
+The full listing of supported parameters can be found in the [`teleport-access-graph` Helm chart reference](../../reference/helm-reference/teleport-access-graph.mdx)
+
+Teleport Access Graph is enabled by default for Teleport Team users.
+You can skip the setup process here and start using Access Graph right away
+by signing up for a [Teleport Team trial](https://goteleport.com/signup/).
+
+## Prerequisites
+
+- Kubernetes >= v1.21
+- Helm >= (=helm.version=)
+- A running Teleport Enterprise cluster v14.3.4 or later.
+  - For the purposes of this guide, we assume that the Teleport cluster is set up
+    [using a Helm chart](../../deploy-a-cluster/helm-deployments.mdx)
+    in the same Kubernetes cluster that will be used to deploy Teleport Access Graph .
+- An updated `license.pem` with Teleport Policy enabled.
+- A PostgreSQL database server v14 or later.
+  - Access Graph needs a dedicated [database](https://www.postgresql.org/docs/current/sql-createdatabase.html) to store its data.
```

Comment:
I would be clearer about what this is linking to: does this mean it requires Postgres? Would it make sense to say we require Postgres without including the link?

Reply:
I link to the `CREATE DATABASE` docs because this is my attempt to clarify the overloaded "database" terminology.

In general, "database" can refer to a database server / cluster, i.e. if you `kubectl run postgres`, you have "a postgres database" deployed. But inside a single Postgres server / cluster there can be several "databases", i.e. separate silos for data (the hierarchy being roughly: database -> schema -> table).

In the TAG case, we need a separate database in the Postgres hierarchy sense, i.e. you should not colocate TAG in the same database as your business app (but it *can* be on the same Postgres server / cluster!).

I'd guess that no serious user would try to comingle TAG and a different app in the same Postgres database (in the hierarchy sense), but I tried to clarify this just to be safe 😅